includes/multi-currency/CountryFlags.php

```diff
@@ -288,12 +289,12 @@ public static function get_by_country( string $country ): string {
 	 */
 	public static function get_by_currency( string $currency ): string {
 		$exceptions = [
-			'ANG' => '',
-			'BTC' => '',
-			'XAF' => '',
-			'XCD' => '',
-			'XOF' => '',
-			'XPF' => '',
+			Currency_Code::NETHERLANDS_ANTILLEAN_GUILDER => '',
+			Currency_Code::BITCOIN                       => '',
```

Comment:
@jessy-p @zmaglica should we keep it, not sure why was it there in the first place?

Reply:
Looks like its used for the Multicurrency tab on Admin page, its fine to leave it there, I suppose, since its already there.